notebooks/17_foundations__stratified_spaces.ipynb

```diff
@@ -61,19 +61,16 @@
     }
```

Comment:
<p>In different algorithms related to graphs, we might require to ... (see typos)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1608/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>